### PR TITLE
Navigation ho collapse

### LINKED ISSUE #34: Improve navigation bar
We need to improve  the navigation bar

### PR DESCRIPTION
**Pull request by marobo**
close  #34 

### Vizaun
daudaun nee ita nia situs sei uza hela front-end direta ho online no seidauk suporta offline, no mos collapse ba parte header nian seidauk furak.

### Kondisaun no Satisfasaun
- hatama bootstrap 
- hatama jquery js
- hadia jeitu husi parte header nian liliu collapse ba suporta laiar kiik sira.

### Oinsa nee atu la'o
- run `yarn` atu install package jquery
- visita situs no inspect ba laiar kiik hodi haree jeitu husi collapse parte header nian

### CODE REVIEW

**Comment by eraulo** [comment]
@marobo ita  bele hadia fali nia header tamba wainhira ita hare iha screen kiik ita labele hare nia menu sira ne'ebe iha header ne'e. Exemplo: `Galaxy S2`
Screenshot from 2019-08-03 15-47-17

File: pykafe/pykafe/static/css/navbar.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+ul.navbar-nav {
```
Comment (eraulo) [inline pykafe/pykafe/static/css/navbar.css]: Presisa hamos tiha navbar.css tamba iha pykafe.css iha tiha ona.

File: pykafe/pykafe/templates/base.html [hunk 2]
```diff
@@ -21,10 +21,16 @@
...
+        <link type="text/css" rel="stylesheet" href="{% static 'bootstrap/dist/css/bootstrap.min.css' %}" />
+
+        <!-- Optional JavaScript -->
+        <script type="text/javascript" src="{% static 'jquery/dist/jquery.min.js' %}"></script>
+        <script type="text/javascript" src="{% static 'bootstrap/dist/js/bootstrap.min.js' %}"></script>
+        <script type="text/javascript" src="{% static 'popper.js/dist/umd/popper.min.js' %}"></script>
 
         {# Global stylesheets #}
         <link rel="stylesheet" type="text/css" href="{% static 'css/pykafe.css' %}">
+        <link rel="stylesheet" type="text/css" href="{% static 'css/navbar.css' %}">
```
Comment (eraulo) [inline pykafe/pykafe/templates/base.html]: Hamos tiha lina link navbar.css

File: pykafe/pykafe/templates/base.html [hunk 3]
```diff
@@ -48,11 +54,8 @@
 
         {% include 'includes/footer.html' %}
 
-	       <!-- Optional JavaScript -->
-        <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+        <!-- jQuery -->
 	       <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
```
Comment (eraulo) [inline pykafe/pykafe/templates/base.html]: Tamba sa mak jquery-3.3.1.slim.min.js ls iha local?

File: pykafe/pykafe/templates/includes/header.html [hunk 4]
```diff
@@ -2,34 +2,38 @@
 {% load wagtailcore_tags %}
 
 <nav class="navbar navbar-expand-lg header">
-  {% get_site_root as home %}
-  <a class="navbar-brand" href="{% pageurl home %}">
-    {% include 'includes/logo_colours.html' %}
-  </a>
 
+     <div class="container">
```
Comment (eraulo) [inline pykafe/pykafe/templates/includes/header.html]: Tamba sa mak `div` iha fali `nav` nia laran ne'e ita bele hare lai regulamentu html nia iha kraik ne'e:
`<div> <nav> links 1</nav> </div>`
https://stackoverflow.com/questions/18628097/whats-the-difference-between-using-nav-and-div-around-bootstrap-3-navbars

**Comment by marobo** [comment]
> @marobo ita bele hadia fali nia header tamba wainhira ita hare iha screen kiik ita labele hare nia menu sira ne'ebe iha header ne'e. Exemplo: `Galaxy S2`
> Screenshot from 2019-08-03 15-47-17

Hi @eraulo iha hau nian bele hare'e hela, karik ita bele  check fali ita nian no zoom ba 100%

File: pykafe/pykafe/templates/includes/header.html (hunk 4 above)
Comment (marobo) [reply pykafe/pykafe/templates/includes/header.html]: hi @eraulo , hau hanoin nee hanesan deit, depende ita uza.

File: pykafe/pykafe/static/css/navbar.css (hunk 1 above)
Comment (marobo) [reply pykafe/pykafe/static/css/navbar.css]: hamos ona!

File: pykafe/pykafe/templates/base.html (hunk 2 above)
Comment (marobo) [reply pykafe/pykafe/templates/base.html]: hamos ona!

**Comment by MarioAP** [comment]
Nara Pykafe iha header nee presija muda oitoan tan ba liur tamba tama liu tia ba laran

**Pull request merged**